### DISCUSSION TITLE
百度网盘挂载后下载提示hit black uid file

### DISCUSSION THREAD

**Discussion by fyr611** [AlistGo/alist#1885] [Q&A]
如题，挂载后，目录可以正常显示。点击下载提示{"errno":9013,"errmsg":"hit black uid file","request_id":8729538780664937413}
用新账号测试也是同样问题，所以排除账号被封的可能。

**Comment by ampcr** [comment]
同遇..有点尬, 感觉是百度网盘也升级了策略.
半年前跟阿里云负载均衡，大文件一直没有问题，最近俩天失效了。
于是到V3也一样这个提示, 我知道撒请求头可以解决, 但是不是访问的所有人都会请求头的.
